.github/blueprints/blueprint.json

```
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "/wp-admin/admin.php?page=wp-ai-client",
```

This page doesn't exist, should be:
```suggestion
  "landingPage": "/wp-admin/options-general.php?page=ai-experiments",
```